docker-compose.yml

```diff
@@ -28,6 +28,9 @@ services:
       # You can set these volumes to point to empty directories on host
       - ./tests/data/embeddings/:/data/embeddings/
       - ./tests/data/models/:/data/models/
-
+    deploy:
+      resources:
+        limits:
+          memory: 8GB
```

Comment:
1. You're right! It should be `8g`.
2. You're correct, this would not help with OOM issues, but it could help us with keeping the container size down during local development to avoid these issues in the future.

Reply:
I see, so as to keep container size below 8g to prevent OOM exception on Github cloud build